core/array.cpp

```diff
@@ -413,7 +413,11 @@ Array::Array() {
 	_p->refcount.init();
 }
 
+<<<<<<< HEAD
```

Comment:
Failed merge markers!

Reply:
In the last commit, every merge conflict is resolved and no markers were left.